docs/libertyDev.md

```diff
@@ -80,3 +82,58 @@ test {
+
+Start a Liberty server in a local container using the Dockerfile that you provide. An alternative to the `libertyDevc` task is to specify the `libertyDev` task with the `--container` option. 
+
+The Dockerfile must copy the application .war file and the server configuration files that the application requires into the container. A sample Dockerfile is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits the container is stopped and deleted and the logs are preserved as described below.
+
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Gradle task. You can examine the commands used to build and run the container by viewing the console output of dev mode.
+
+When dev mode runs with container support it still provides the same features. It monitors files for changes and runs tests either automatically or on demand. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. build/wlp/usr/servers/defaultServer/logs.
+
+This mode publishes the container ports 9080, 9443 and 7777 by default. If your application needs to publish other ports add them to the `dockerRunOpts` option either in the `build.gradle` file or on the `gradle` command line. E.g. `--dockerRunOpts="-p 9081:9081"`
+
+### Limitations
+
+For the current technology preview, the following limitations apply.
+
+- Platform limitations:
+  - Supported on macOS and Windows with Docker Desktop installed. Not supported on Linux at the moment.
+
+- Dockerfile limitations:
+  - The Dockerfile must copy only one .war file for the application.  Other application archive formats or multiple .war files are not supported.
+  - Hot deployment is only supported for individual configuration files that are specified as the source in the Dockerfile's COPY commands. Hot deployment is not supported for COPY commands with variable substitution, wildcard characters, spaces in paths, paths relative to WORKDIR, multi-stage builds, or entire directories specified as the source.
+
+### Examples
+
+Start dev mode with the server in a container using the Dockerfile in the project root.
+```
+$ gradle libertyDevc
+```
+
+Customizing the container configuration using [dev extension properties](libertyExtensions.md#dev-extension-properties) in `build.gradle`.  Note that changing these while dev mode is running is not supported.
+```
+liberty {
+    dev {
+        container = true
+        dockerRunOpts = '-p 9081:9081'
+        dockerfile = file('myDockerfile')
+    }
+}
+```
+
+### Command Line Parameters
+
+These parameters are available in addition to the ones in the `libertyDev` section above.
+
+| Parameter | Description | Required |
```

Comment:
Use the following header/columns like all the other tasks:

| Attribute | Type | Since | Description | Required |

Reply:
Fixed